### ISSUE TITLE
Exception when starting 0.0.33

### ISSUE THREAD

**Issue by Apollon77** [soef/ioBroker.alexa#13]
```
2018-03-19 22:29:07.003  - info: alexa.0 starting. Version 0.0.33 in /opt/iobroker/node_modules/iobroker.alexa, node: v6.13.1
2018-03-19 22:29:08.264  - error: alexa.0 uncaught exception: Cannot read property 'length' of undefined
2018-03-19 22:29:08.265  - error: alexa.0 TypeError: Cannot read property 'length' of undefined
    at doIt (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:112:30)
    at AlexaRemote.Alexa.updateStates (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:134:7)
    at devices.update (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:317:18)
    at safeCallback (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:188:13)
    at doit (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:247:24)
    at safeCallback (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:188:13)
    at Devices.setStateEx (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:702:17)
    at /opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:765:22
    at doit (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:249:13)
    at safeCallback (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:188:13)
2018-03-19 22:29:08.306  - error: Caught by controller[0]: TypeError: Cannot read property 'length' of undefined
2018-03-19 22:29:08.306  - error: Caught by controller[0]:     at doIt (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:112:30)
2018-03-19 22:29:08.307  - error: Caught by controller[0]:     at AlexaRemote.Alexa.updateStates (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:134:7)
2018-03-19 22:29:08.307  - error: Caught by controller[0]:     at devices.update (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:317:18)
2018-03-19 22:29:08.307  - error: Caught by controller[0]:     at safeCallback (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:188:13)
2018-03-19 22:29:08.307  - error: Caught by controller[0]:     at doit (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:247:24)
2018-03-19 22:29:08.308  - error: Caught by controller[0]:     at safeCallback (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:188:13)
2018-03-19 22:29:08.308  - error: Caught by controller[0]:     at Devices.setStateEx (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:702:17)
2018-03-19 22:29:08.308  - error: Caught by controller[0]:     at /opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:765:22
2018-03-19 22:29:08.308  - error: Caught by controller[0]:     at doit (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:249:13)
2018-03-19 22:29:08.308  - error: Caught by controller[0]:     at safeCallback (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:188:13)
2018-03-19 22:29:08.309  - error: host.nuc2 instance system.adapter.alexa.0 terminated with code 0 (OK)

```

**Comment by soef** [comment]
Can you please try Version 0.0.35?

**Comment by Apollon77** [comment]
Current github ... hm ... it is not crashing directly on startup but it also seems so that states are not updated on startup.
And on histpry-Trigger I get

```
2018-03-26 23:12:29.526  - error: alexa.0 uncaught exception: callback is not a function
2018-03-26 23:12:29.527  - error: alexa.0 TypeError: callback is not a function
    at AlexaRemote.Alexa.updateStates (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:115:55)
    at refreshTimer.set (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:56:15)
    at Timeout._onTimeout (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:1519:13)
    at ontimeout (timers.js:386:11)
    at tryOnTimeout (timers.js:250:5)
    at Timer.listOnTimeout (timers.js:214:5)
2018-03-26 23:12:29.530  - info: alexa.0 terminating
2018-03-26 23:12:29.567  - error: Caught by controller[1]: TypeError: callback is not a function
2018-03-26 23:12:29.568  - error: Caught by controller[1]:     at AlexaRemote.Alexa.updateStates (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:115:55)
2018-03-26 23:12:29.568  - error: Caught by controller[1]:     at refreshTimer.set (/opt/iobroker/node_modules/iobroker.alexa/alexa.js:56:15)
2018-03-26 23:12:29.569  - error: Caught by controller[1]:     at Timeout._onTimeout (/opt/iobroker/node_modules/iobroker.alexa/node_modules/soef/soef.js:1519:13)
2018-03-26 23:12:29.569  - error: Caught by controller[1]:     at ontimeout (timers.js:386:11)
2018-03-26 23:12:29.569  - error: Caught by controller[1]:     at tryOnTimeout (timers.js:250:5)
2018-03-26 23:12:29.569  - error: Caught by controller[1]:     at Timer.listOnTimeout (timers.js:214:5)
```

**Comment by soef** [comment]
Should be fixed in Version 0.0.36

**Comment by Apollon77** [comment]
Works now but I needed to manually delete the cookie text (i never entered)  in adapter config
